aioredis/cluster/cluster.py

```diff
@@ -490,15 +488,14 @@ def __init__(self, nodes, db=0, password=None, encoding=None,
         self._maxsize = maxsize
         self._cluster_pool = {}
 
-    def get_nodes_entities(self):
+    def get_nodes_entities(self, **kwargs):
         return self._cluster_pool.values()
 
-    @asyncio.coroutine
-    def get_cluster_pool(self):
+    async def get_cluster_pool(self):
         cluster_pool = {}
         nodes = list(self._cluster_manager.masters)
         tasks = [
-            async_task(create_pool(
+            asyncio.ensure_future(create_redis_pool(
```

Comment:
This ensure_future seems to be unnecessary, as the tasks are awaited in the gather just below.